README.md

```diff
@@ -20,9 +20,9 @@ docker-compose run web ./manage.py migrate
 ```
 
 ### Loading data
-1) `docker-compose run web ./manage.py shell`
+1) `docker-compose run web ./manage.py shell` -- We normally create ./scripts/console to standardize this pattern
 ```
 from philly import load
```

Comment:
We often encapsulate this type of task into a management command, and we usually have it run automatically via `./scripts/setup` so that the developer doesn't need to manually populate data after initial project setup. But that's not always feasible, as with Cicero, so this isn't consistent across projects.